.github/workflows/cuda.yml

```diff
@@ -73,7 +90,7 @@ jobs:
             GITHUB_ACTIONS=${{ env.github_actions }}
             OS_NAME=${{ env.os_name }}
             COMPILER=${{ matrix.compiler }}
-            TASK=${{ env.task }}
+            TASK=${{ matrix.tree_learner }}
```

Comment:
I can't make a suggestion for that line, so please use `tree_learner` in the job name to allow to distinguish CUDA CI jobs from CUDA EXP CI jobs.
```
name: ${{ matrix.tree_learner }} ${{ matrix.cuda_version }} ${{ matrix.method }} (linux, ${{ matrix.compiler }}, Python ${{ matrix.python_version }})
```
https://github.com/microsoft/LightGBM/blob/1d8bfd8c9b3fa7d56db81201adedb61cfc11669c/.github/workflows/cuda.yml#L19

Reply:
Resolved via [b008424](https://github.com/microsoft/LightGBM/pull/4630/commits/b0084249634c595bf895173df557eecbc39e00c7)